topics/epigenetics/tutorials/ewas-suite/tutorial.md

```diff
@@ -191,7 +183,7 @@ However, for the purpose of this tutorial we would like you to upload phenotype
 In addition to downstream analysis users can annotate the differentially methylated loci at the promoter regions of genes with gene function descriptions, and relationships between these concepts.
 
 > <hands-on-title> Annotate Differentially Methylated Position</hands-on-title>
-> 1. Run **chipeakanno annopeaks** {% icon tool %}on the output of minfi_dmp with the following parameters
+> 1. Run **ChIPpeakAnno annoPeaks** {% icon tool %}on the output of minfi_dmp with the following parameters
```

Comment:
If you could use the new tool linking syntax it would be really nice for users!

We have a [guide on how to do it](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#tool-links), but in short just switch to `{% tool [Your Tool](toolshed.g2.bx.psu.edu/repos/..../your-tool/1.0.0+galaxy0) %}`

(this comment applies to several other places in the tutorial as well.)